### PR TITLE
Add example for duration operator

### PR DESCRIPTION
**Pull request by treydock**
Signed-off-by: Trey Dockendorf <tdockendorf@osc.edu>

Goes with documentation from here: https://github.com/kyverno/website/pull/241

For this feature: https://github.com/kyverno/kyverno/pull/2214

### CODE REVIEW

File: other/pod_lifetime_annotation/policy.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: pod-lifetime
+  annotations:
+    policies.kyverno.io/title: Enforce pod duration
+    policies.kyverno.io/category: Sample
+    policies.kyverno.io/minversion: 1.4,2
+    policies.kyverno.io/subject: Pod
+    policies.kyverno.io/description: >-
```
Comment (chipzoller) [inline other/pod_lifetime_annotation/policy.yaml:10]: See sister comment on your kyverno/website PR, but one other thing we're trying to standardize on with this `description` annotation is to state two things: 

1. Why is such a policy valuable?
2. What does this policy do?

You cover the second one but if you would please put one or two short sentences here for why this policy is useful.

**Review by chipzoller — CHANGES_REQUESTED** [review]
One update, if you please, to the description annotation.

File: other/pod_lifetime_annotation/policy.yaml (hunk 1 above)
Comment (treydock) [reply other/pod_lifetime_annotation/policy.yaml:10]: Added some extra description to cover why this is valuable

File: other/pod_lifetime_annotation/policy.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: pod-lifetime
+  annotations:
+    policies.kyverno.io/title: Enforce pod duration
+    policies.kyverno.io/category: Sample
+    policies.kyverno.io/minversion: 1.4,2
```
Comment (chipzoller) [inline other/pod_lifetime_annotation/policy.yaml:8]: comma between "4" and "2" rather than period

Comment (treydock) [reply other/pod_lifetime_annotation/policy.yaml]: Fixed the typo

File: other/pod_lifetime_annotation/policy.yaml (hunk 1 above)
Comment (chipzoller) [reply other/pod_lifetime_annotation/policy.yaml:10]: "This validation is valuable with annotations are used..."

Should this be "when annotations are used"?

Comment (treydock) [reply other/pod_lifetime_annotation/policy.yaml:10]: Fixed wording

**Review by chipzoller — APPROVED** [review]

**Pull request merged**